include/mapbox/geometry/feature.hpp

```diff
@@ -13,7 +13,7 @@ namespace mapbox { namespace geometry {
 
 struct value;
 
-using value_base = mapbox::util::variant<bool, int64_t, uint64_t, double, std::string,
+using value_base = mapbox::util::variant<std::nullptr_t, bool, int64_t, uint64_t, double, std::string,
```

Comment:
@mourner - `std::nullptr_t` is problematic in this context^  As @kkaefer discovered visitor code might attempt to construct `std::string(nullptr)` leading to runtime segfault. It'll be better to use custom empty(null) type e.g

```c++
struct null_value {} ;
```